### DISCUSSION TITLE
How to disable e-mail 2FA?

### DISCUSSION THREAD

**Discussion by beedell-roke** [padloc/padloc#699] [Q&A] [answered]
My e-mail account's password is stored in my password manager, so if I am to switch to Padloc, I either need to set my e-mail account's password to something that I'll remember (which is undesirable because it's insecure and inconvenient) lest I forget and become locked-out of my vault, or I shall need to disable e-mail 2FA.

Apologies if this is trivial to do; I didn't see the option when I last checked.

**Comment by MaKleSoft** [comment, marked as answer]
Hi there! You can set up alternative MFA methods under Settings / Security. Padloc current supports hardware tokens (e.g. Yubikey) or TOTP (e.g. via Google Authenticator). Once you've set up an alternative MFA method, that one will be used instead of email by default (email still functions as a fallback in case you lose access your other MFA method).

**Reply by beedell-roke** [reply]
I can't do that though, because I don't have a subscription.
